var/spack/repos/builtin/packages/py-clang/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyClang(PythonPackage):
+    """Python bindings for clang from clang release branches"""
+
+    # This package is needed for py-tensorflow@2.6. Generally speaking the
+    # py-libclang package should be used instead. It not recommended to add
+    # updated versions to this package. Note that no llvm dependency is
+    # specified here because llvm-5 will cause spack concretization to use an
+    # equally old GCC.
```

Comment:
Will this package work if no LLVM is installed on the system anywhere?

Reply:
It is a shame that rebasing causes comments to be lost. There was an unresolved conversation around this where I had an open question about how to handle this dependency. There are two things that could be done.

1. leave the dependency in place so people could use the bindings if they had llvm-5 available.
2. remove this dependency for this version of TF and mask it in the setup.py file.

I chose to do (1) for now simply because the package was already in the PR and it would be easier to flip (1) to (2) than it would be to flip (2) to (1).

Reply:
Could we keep LLVM as a dep? This only affects one specific version of TF, right?

Reply:
Yes, but as I tried to convey in my comment, that will then cause the DAG to have a really old compiler version. It seems that would cause problems for people as well. At least at my site, it would make TF-2.6.0 undeployable, particularly since it involves python.

Reply:
My preferred path is option (2) above. Like I said, it was an open question and I structured it to be flexible. The clang dependency was new in TF-2.6 and changed to the other clang project in TF-2.7. As far as I know, the clang bindings are not strictly needed and for those who need them, TF-2.7 is available.

Reply:
I removed, and masked, the py-clang dependency for TF-2.6. That is what felt best to me, especially since TF-2.7 is using the newer py-libclang package.